python/pyngraph/ops/fused/clamp.cpp

```diff
@@ -17,14 +17,15 @@
 #include <pybind11/pybind11.h>
```

Comment:
It is weird that git is confused and threat `clamp.cpp` as new version of `elu.cpp`.
Is that possible that you removed `elu.cpp` by accident?

Reply:
There is also a possibility that you forgot to add `elu.cpp` after moving this file to a new location.

Reply:
There is also a possibility that you forgot to add elu.cpp after moving this file to a new location.

Reply:
Accidentally, I converted the original file to `clamp.cpp` instead of its copy. However, the copy file is in right place and works well.